src/app/features/languageSlice.ts

```diff
@@ -6,10 +6,12 @@ interface ILanguageState {
     currentLanguage: LanguageCode;
 }
 
+const localStorageLanguage = localStorage.getItem('language');
```

Comment:
can we rename this variable ?localStorageLanguage

Reply:
Yes i will change it